src/index.js

```diff
@@ -1 +1,57 @@
-import './style.css';
\ No newline at end of file
+import './style.css';
+
+const scores = 'https://us-central1-js-capstone-backend.cloudfunctions.net/api/games/7oyqKQQCxsTaogFAwN0N/scores/';
+
+let scoresArray = [];
+
+const scoresList = document.getElementById('scoresList');
+
+const refreshButton = document.getElementById('refresh');
+
+function displayScores() {
```

Comment:
- [ ] <b>REQUIRED:<b/> Prefer using ES6 arrow function. Actually, there is nothing wrong with ES5 functions syntax but it is mandatory to use es6 features in the javascript module. besides, Arrow functions provide a simplified structure, allowing for a more concise way to write function expressions. which makes them better than ES5 functions(function declarations). Here is the syntax of an ES6 arrow function...
```javascript
  const functionName = () => {
     // code here
  }
```